streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KStream.scala

```diff
@@ -320,9 +320,9 @@ class KStream[K, V](val inner: KStreamJ[K, V]) {
    * @return a [[KStream]] that contains records with unmodified key and new values (possibly of different type)
    * @see `org.apache.kafka.streams.kstream.KStream#transformValues`
    */ 
-  def transformValues[VR](valueTransformerWithKeySupplier: ValueTransformerWithKeySupplier[K, V, VR],
+  def transformValues[VR](valueTransformerSupplier: ValueTransformerWithKeySupplier[K, V, VR],
```

Comment:
Why this renaming?

Reply:
Intellij madness that I missed. Fixing...

Reply:
Ah, no, not madness - I changed it to match the scala doc parameter name.

Reply:
Maybe we should update the doc parameter name instead?

Reply:
Sure, I just prefer short param names.